include/verilated_funcs.h

```diff
@@ -519,18 +522,19 @@ static inline void VL_ASSIGNBIT_WO(int bit, WDataOutP owp) VL_MT_SAFE {
 static inline WDataOutP VL_EXTEND_WI(int obits, int, WDataOutP owp, IData ld) VL_MT_SAFE {
     // Note for extracts that obits != lbits
     owp[0] = ld;
-    for (int i = 1; i < VL_WORDS_I(obits); ++i) owp[i] = 0;
+    VL_MEMSET_W(owp + 1, 0, VL_WORDS_I(obits) - 1);
     return owp;
 }
 static inline WDataOutP VL_EXTEND_WQ(int obits, int, WDataOutP owp, QData ld) VL_MT_SAFE {
     VL_SET_WQ(owp, ld);
-    for (int i = VL_WQ_WORDS_E; i < VL_WORDS_I(obits); ++i) owp[i] = 0;
+    VL_MEMSET_W(owp + VL_WQ_WORDS_E, 0, VL_WORDS_I(obits) - VL_WQ_WORDS_E);
     return owp;
 }
 static inline WDataOutP VL_EXTEND_WW(int obits, int lbits, WDataOutP owp,
                                      WDataInP const lwp) VL_MT_SAFE {
-    for (int i = 0; i < VL_WORDS_I(lbits); ++i) owp[i] = lwp[i];
-    for (int i = VL_WORDS_I(lbits); i < VL_WORDS_I(obits); ++i) owp[i] = 0;
+    const int lwords = VL_WORDS_I(lbits);
+    VL_MEMCPY_W(owp, lwp, lwords);
+    VL_MEMSET_W(owp + lwords, 0, VL_WORDS_I(obits) - lwords);
     return owp;
```

Comment:
```suggestion
    VL_MEMSET_W(owp + lwords, 0, VL_WORDS_I(obits) - lwords);
    return VL_MEMCPY_W(owp, lwp, lwords);
```

I suspect this will typically be equal or faster as cache misses for reading the lwp can be hidden under the zero.